src/phoenix/session/session.py

```diff
@@ -343,14 +354,14 @@ def close_app() -> None:
     logger.info("Session closed")
 
 
-def _get_url(port: int, is_colab: bool) -> str:
+def _get_url(host: str, port: int, is_colab: bool) -> str:
     """Determines the IFrame URL based on whether this is in a Colab or in a local notebook"""
     if is_colab:
         from google.colab.output import eval_js  # type: ignore
 
         return str(eval_js(f"google.colab.kernel.proxyPort({port}, {{'cache': true}})"))
 
-    return f"http://localhost:{port}/"
+    return f"http://{host}:{port}/"
```

Comment:
I am thinking we should just resolve to localhost here. Thoughts?

Reply:
(rather than display http://0.0.0.0:6060/ and have the user be using that link.

Reply:
we should just let it be because at that point the user knows more than us about what they are doing